### PR TITLE
Refactor common errors to rxutils, split i18n to seperate module

### PR DESCRIPTION
**Pull request by natechadwick**

### CODE REVIEW

File: projects/sitemanage/src/main/resources/Rhythmyx/sys_resources/ant/copy-resources.xml [hunk 1]
```diff
@@ -109,7 +109,7 @@
                password="${perc.ftp.password}"
 			   useFtps="${perc.ftp.useFtps}"
 			   passive="yes"
-               skipFailedTransfers="true"
+               skipFailedTransfers="false"
```
Comment (sonyDeswal) [inline projects/sitemanage/src/main/resources/Rhythmyx/sys_resources/ant/copy-resources.xml:112]: This is intended change?

File: system/services/src/com/percussion/services/assembly/impl/nav/PSNavHelper.java [hunk 2]
```diff
@@ -881,7 +881,7 @@ public PSLocator findNavon(PSLocator folderLoc)
          return null;
       }
 
-      if (folderLoc!= null && folderLoc.getId()==1) {
+      if (folderLoc.getId() == 1) {
```
Comment (sonyDeswal) [inline system/services/src/com/percussion/services/assembly/impl/nav/PSNavHelper.java:884]: Can throw an NPE now...

**Review by sonyDeswal — COMMENTED** [review]
Just had 2 comments to verify if changes are intentional.. rest looks good

File: projects/sitemanage/src/main/resources/Rhythmyx/sys_resources/ant/copy-resources.xml (hunk 1 above)
Comment (natechadwick) [reply projects/sitemanage/src/main/resources/Rhythmyx/sys_resources/ant/copy-resources.xml:112]: Yes

File: system/services/src/com/percussion/services/assembly/impl/nav/PSNavHelper.java (hunk 2 above)
Comment (natechadwick) [reply system/services/src/com/percussion/services/assembly/impl/nav/PSNavHelper.java:884]: It is impossible for folderLoc to be null at that location.  line 877 checks for null and exits the function.

**Pull request merged**